docs/book/supported_features_matrix.md

```diff
@@ -2,30 +2,32 @@
+| K8s Multi-node Control Plane support                                                          | v1.0.3                                     | v2.1.0                                                 | ✅              | ✅              | ✅            | ✅          |
+| [WaitForFirstConsumer](https://kubernetes.io/docs/concepts/storage/storage-classes/) (beta)   | v1.0.3                                     | v2.1.0                                                 | ✅              | ✅              | ✅            | ✅          |
+| Offline Volume Expansion support (beta) (Block Volume only)                                   | v2.0.0                                     | v2.1.0                                                 | ✅              | ✅              | ✅            | ❌          |
+| Encryption support via VMcrypt (Block Volume only)                                            | v2.0.0                                     | v2.1.0                                                 | ✅              | ✅              | ✅            | ❌          |
+| Dynamic File PV support through vSAN 7.0 File Services on vSAN Datastores                     | v2.0.0                                     | v2.1.0                                                 | ✅              | ✅              | ✅            | ❌          |
+| In-tree vSphere volume migration to CSI (beta)                                                | v2.1.0                                     | v2.1.0                                                 | ✅              | ✅              | ❌            | ❌          |
+| Online Volume Expansion support (beta) (Block Volume only)                                    | v2.2.0                                     | v2.2.0                                                 | ✅              | ❌              | ❌            | ❌          |
 
 _Notes_:
 
 * We do not recommend `beta` features for production use. Subsequent releases may introduce incompatible changes.
-* vSphere CSI driver is not supported on Windows based vCenter.
-* vSphere CSI driver is not supported on vSAN stretch cluster.
-* vSphere CSI driver and Cloud Native Storage in vSphere does not currently support Storage DRS feature in vSphere.
-* The dynamic creation of read-write-many (`RWX`) and read-only-many (`ROX`) file based Persistent Volumes is only available in vSphere 7.0 through vSAN 7.0 File Services.
-* PV Encryption, offline volume expansion and Topology/AZ support are only available on block based Persistent Volumes. These features are not available with PVs backed by file shares.
-* Offline volume expansion requires a minimum Kubernetes version of 1.16+.
+* vSphere CSI driver does not support Windows based vCenter.
+* vSphere CSI driver currently does not support vSAN stretch cluster.
+* vSphere CSI driver and Cloud Native Storage does not currently support Storage DRS feature in vSphere.
```

Comment:
Is this still the case? We had customers saying they tested it and it works?

Reply:
`Storage DRS is not supported for FCD volumes.` There are some cases when the volume is not attached to the VM, storage DRS does not work for FCD volumes.